.github/workflows/app-etterlatte-hendelser-joark.yaml

```diff
@@ -0,0 +1,57 @@
+name: etterlatte-hendelser-joark
+
+on:
+  workflow_dispatch: # Allow manually triggered workflow run
+    inputs:
+      deploy-prod:
+        description: 'Deploy til produksjon'
+        required: false
+        default: 'false'
+        type: choice
+        options:
+          - true
+          - false
+  push:
+    branches:
+      - main
+    paths:
+      - apps/etterlatte-hendelser-joark/**
+      - libs/saksbehandling-common/**
+      - libs/etterlatte-kafka/**
+      - libs/etterlatte-ktor/**
+      - libs/etterlatte-oppgave-model/**
+      - libs/ktor2client-auth-clientcredentials/**
+      - gradle/libs.versions.toml
+  pull_request:
+    branches:
+      - main
+    paths:
+      - apps/etterlatte-hendelser-joark/**
+      - libs/saksbehandling-common/**
+      - libs/etterlatte-kafka/**
+      - libs/etterlatte-ktor/**
+      - libs/etterlatte-oppgave-model/**
+      - libs/ktor2client-auth-clientcredentials/**
+      - gradle/libs.versions.toml
+
+jobs:
+  test:
+    if: github.event_name == 'pull_request'
+    uses: ./.github/workflows/.test.yaml
+    secrets: inherit
+
+  build:
+    if: github.event_name != 'pull_request'
+    uses: ./.github/workflows/.build.yaml
+    secrets: inherit
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+
+#  deploy:
```

Comment:
Kommenter inn?

Reply:
Appen er ikke klar for å lese og behandle hendelser riktig enda, så avventer litt.

Reply:
Jo, brainfart... dette skulle vært med... drar en spansk en rett i main. Glemte at vi skiller på dev/prod i _deploy_, ikke selve workflowen.